pyanaconda/modules/network/initialization.py

```diff
@@ -232,6 +232,10 @@ def _run(self, nm_client):
 
             iface = device.get_iface()
 
+            if is_nbft_device(iface or ""):
```

Comment:
This totally work, I'm just thinking that:
```python
if iface and is_nbft_device(iface):
```
wouldn't be better to read.

No need to update the code, just saying for future I guess :D :D